### ISSUE TITLE
Useful code for chairs

### ISSUE THREAD

**Issue by ldlasso2** [christopher-beckham/hologan-pytorch#9]
Hello Cristopher, your code is amazing! I had a question and it is if it is possible to do 3D recreation with chairs or with cats, since in your examples there are only celebrities and cars. Do you have to make adjustments?

**Comment by christopher-beckham** [comment]
Hi,

Thank you! Yes it should be possible, assuming your datasets have a decent amount variation in both camera pose and object identity so that it can leverage the inductive bias of the model architecture (since you want to try and disentangle those two things). Of course, hyperparameter tuning will always need to be done on a case-by-case basis for each dataset.

With regard to hyperparameters, this may also be useful to you: https://github.com/christopher-beckham/hologan-pytorch/issues/8

Thanks.

**Comment by juanescendales** [comment]
Hi Cristopher, I'm trying to replicate the results of the paper with chairs too with your nice code with this configuration:

 --lamb=25.0 --angles="[0,0,-90,90,0,0]" --beta1=0.5 --beta2=0.999 --update_g_every=1 --ngf=512 --ndf=64

This configuration is the same that you had in the example of the cars so I take this one as my starting point, after training 200 epochs this was the result:

https://user-images.githubusercontent.com/26862668/102814451-ab1fa580-4398-11eb-9251-fd8919cf2095.mp4

As you can tell, instead of the expected "yaw rotation", that was not achieved, I had that strange result. My data set had many .png 64x64 images of several different kinds of chairs in 8 poses among the horizontal axis, my question is if you consider that I need to have more poses of those chairs or there is something wrong in the configuration. 

Thank you very much.

PD:
![3](https://user-images.githubusercontent.com/26862668/102815669-b83d9400-439a-11eb-90fb-1158bfd60f27.png)
![4](https://user-images.githubusercontent.com/26862668/102815672-b8d62a80-439a-11eb-9772-a693bebd5006.png)
![5](https://user-images.githubusercontent.com/26862668/102815673-b8d62a80-439a-11eb-8731-f84f669adf4c.png)
![6](https://user-images.githubusercontent.com/26862668/102815674-b8d62a80-439a-11eb-9ea7-30be918d04a3.png)
![7](https://user-images.githubusercontent.com/26862668/102815675-b8d62a80-439a-11eb-9d5e-de91054c9493.png)
![8](https://user-images.githubusercontent.com/26862668/102815677-b96ec100-439a-11eb-864b-0efc9e21934a.png)
![1](https://user-images.githubusercontent.com/26862668/102815678-b96ec100-439a-11eb-96c2-1cc3f37e8678.png)
![2](https://user-images.githubusercontent.com/26862668/102815680-b96ec100-439a-11eb-9c66-d6e1b60ba68e.png)

*This is an example of the poses of one chair, I also have these same poses with other 7000 chairs approx. :)

**Comment by christopher-beckham** [comment]
Hi,

Thanks for sharing! I am actually sort of seeing some rotation in the video, though it appears to be very piecewise. I think based on what you've said about the dataset, you probably do need more variability in the poses, yes.
